content/vendor/guides/configuring-application-snapshots.md

```diff
@@ -0,0 +1,345 @@
+For further details on all of the available specs, please check the Velero documentation [here](https://velero.io/docs/v1.3.2/api-types/backup/)
+
+### Configure Volumes
+
+Volumes by default are not part of the snapshot.
+You configure which volumes to be part of the snaphot by adding the ```backup.velero.io/backup-volumes: ``` label annotation.
+
+In the AppDirect sample application, we have a PostgreSQL stateful set which has the following Volume:
+
+{{< notes title="Note" >}}
+This yaml is trucated for readability purposes. 
+The entire file is located [here](https://github.com/replicatedhq/kotsapps/blob/master/app-direct/manifests/postgres-deployment.yaml) in the GitHub repo
+{{< /notes >}}
+
+```shell
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: app-direct-postgresql
+  labels:
+    app: postgresql
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgresql
+  serviceName: postgres-service
+  template:
+    metadata:
+      labels:
+        app: postgresql   
+    spec:
+      containers:
+...
+      volumeMounts:
+        - name: postgresql-vct
+          mountPath: /var/lib/postgresql/data/pgdata
+          subPath: postgresql-db
+  volumeClaimTemplates:
+  - metadata:
+      name: postgresql-vct
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 2Gi
+
+```
+
+To have the volume included in the snapshot, we add the label annotation as shown below:
+
+```shell
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: app-direct-postgresql
+  labels:
+    app: postgresql
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgresql
+  serviceName: postgres-service
+  template:
+    metadata:
+      labels:
+        app: postgresql 
+      annotations:
+        backup.velero.io/backup-volumes: postgresql-vct # this volume will be included in snapshot
+    spec:
+      containers:
+...
+
+```
+
+The documentation also provides an example [here](https://kots.io/vendor/snapshots/configuring-backup/#identify-volumes) 
+
+### Optional/Advanced Use Case: Backup Hooks
+
+In the example above, we are simply taking a snapshot of the volume. 
+While that may suffice for certain cases, in other cases you may need perform actions before the snapshot takes place. 
+For this, we have Backup Hooks. 
+Velero provides both pre and post [backup hooks](https://velero.io/docs/v1.4/hooks/#docs). 
+These can be configured by adding annotations to the pod itself or in the [Backup spec](https://velero.io/docs/v1.2.0/api-types/backup/). 
+For the purposes of this guide, we'll do the former and use label annotations.
+
+In the example above, a more realistic scenario is that we'll probably want to run ```pg_dump``` and only take a snapshot of that backup. 
+To accomplish this we'll need to:
+
+- Add the ```pre.hook.backup.velero.io/command``` label annotation to execute ```pg_dump``` and put the backup in a new folder.
```

Comment:
Generally for in-line code blocks, I've seen only a single backtick used, e.g.

```
- Add the `pre.hook.backup.velero.io/command` label annotation to execute `pg_dump` and put the backup in a new folder.
```

is there something we gain by using the triple-backtick for inline blocks?